sgx_mm.c

```diff
@@ -445,7 +445,7 @@ int sgx_mm_enclave_pfhandler(const sgx_pfinfo* pfinfo)
     return ret;
 }
 
-int sgx_mm_init(size_t user_base, size_t user_end)
+int sgx_mm_init(size_t user_base, size_t user_end, size_t init_reserve_size)
```

Comment:
BTW, this change seems to break API compatibility. I'm not sure if `sgx_mm_init()` is actually a private or a public API, though it's placed in `emm_private.h` but seems to be needed anyway?

Is this sth. we'd expect?

Reply:
Haitao mentioned that `sgx_mm_init` is a public API and we need avoid break it if possible. The new commit try to address this concern.